lib/compileTransactionName.js

```diff
@@ -0,0 +1,14 @@
+// This file is copy-pasted "as is" from the Dredd Transactions library, where
+// it's also tested. This is a temporary solution,
+// see https://github.com/apiaryio/dredd-transactions/issues/276
+
+
+module.exports = function compileTransactionName(origin) {
+  const segments = [];
+  if (origin.apiName) { segments.push(origin.apiName); }
+  if (origin.resourceGroupName) { segments.push(origin.resourceGroupName); }
+  if (origin.resourceName) { segments.push(origin.resourceName); }
+  if (origin.actionName) { segments.push(origin.actionName); }
+  if (origin.exampleName) { segments.push(origin.exampleName); }
```

Comment:
I understand this is copy-pasted, but you can also do a slightly shorter syntax:

```js
const segments = [
  origin.apiName,
  origin.resourceGroupName,
  origin.resourceName,
  origin.actionName,
  origin.exampleName,
].filter(Boolean).join(' > ')
```

Apart from shorter declaration it also reduces the function's complexity.

Reply:
That's a good suggestion. I'll file it as a DT issue and fix it there. This wile stays copy-pasted and then deleted once DT provides the name directly.

Reply:
https://github.com/apiaryio/dredd-transactions/issues/283